warehouse/ingest-core/src/main/java/datawave/ingest/table/aggregator/GlobalIndexUidAggregator.java

```diff
@@ -99,8 +95,7 @@ public Value aggregate() {
             uids.removeAll(quarantinedIds);
             
             if (!releasedUids.isEmpty()) {
-                if (log.isDebugEnabled())
-                    log.debug("Adding released UIDS");
+                log.debug("Adding released UIDS");
                 uids.addAll(releasedUids);
```

Comment:
I realize this isn't part of your change but couldn't this result in uids exceeding the max threshold?